debug_toolbar/static/debug_toolbar/js/toolbar.timer.js

```diff
@@ -22,17 +22,17 @@
         if (endStat) {
             // Render a start through end bar
             row.innerHTML = '<td>' + stat.replace('Start', '') + '</td>' +
-                            '<td class="djdt-timeline"><div class="djDebugTimeline"><div class="djDebugLineChart"><strong>&#160;</strong></div></div></td>' +
+                            '<td class="djdt-timeline"><svg class="djDebugLineChart" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 100 5" preserveAspectRatio="none"><rect y="0" height="5" fill="#ccc" /></svg></td>' +
```

Comment:
With the move away from HTML, can the following CSS be removed. At first glance, it looks unused now:

```
debug_toolbar/static/debug_toolbar/css/toolbar.css
387  #djDebug span.djDebugLineChart {
434  #djDebug .djDebugInTransaction div.djDebugLineChart strong {
437  #djDebug .djDebugStartTransaction div.djDebugLineChart strong {
440  #djDebug .djDebugEndTransaction div.djDebugLineChart strong {
```

Reply:
Same as with slow. See 4965bf5.